### PR TITLE
Update v10.14 to v10.16 and add events.once to node v10

### PR DESCRIPTION
**Pull request by galkin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://nodejs.org/dist/latest-v10.x/docs/api/events.html#events_events_once_emitter_name>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/node/v11/events.d.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ declare module "events" {
     class internal extends NodeJS.EventEmitter { }
 
     namespace internal {
-        function once(emitter: EventEmitter, event: string | symbol): Promise<any>;
+         function once(emitter: EventEmitter, event: string | symbol): Promise<any>;
```
Comment (Flarna) [inline types/node/v11/events.d.ts]: That's actually not correct.. NodeJs 11 is EOL and was not updated.

**Review by Flarna — APPROVED** [review]

**Review by trivikr — APPROVED** [review]

**Comment by Flarna** [comment]
@galkin Please see also #39116 which does some change regarding `once()` which should be maybe applied here also.

**Comment by galkin** [comment]
@Flarna, thank you. I have joined to #39116

**Comment by galkin** [comment]
@Flarna @trivikr could you merge?

**Comment by Flarna** [comment]
@galkin Sorry, can't do that as I have no push permissions on this repo. This is done by typescript people maintaining the repo. But as PR has the "merge:express" label now it should be integrated soon.

**Pull request merged**